packages/venia-ui/lib/components/AddressBookPage/addressCard.js

```diff
@@ -25,6 +36,12 @@ const AddressCard = props => {
     } = address;
 
     const classes = mergeClasses(defaultClasses, propClasses);
+    const confirmDeleteButtonClasses = {
```

Comment:
Interesting choice of code. Any particular reason why you moved this out of JSX?